samples/simple_app/app.py

```diff
@@ -61,34 +51,55 @@ async def get_webhook_details(domain_name: DomainName) -> WebhookData:
         webhook_secret_key="webhook-secret-key",
     )
 
+async def example_dependency():
+    return "example"
+
 
-async def product_created(payload: Payload, saleor_domain: DomainName):
+async def product_created(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product created!")
     print(payload)
 
 
-async def product_updated(payload: Payload, saleor_domain: DomainName):
+async def product_updated(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product updated!")
     print(payload)
 
 
-async def product_deleted(payload: Payload, saleor_domain: DomainName):
+async def product_deleted(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product deleted!")
     print(payload)
 
 
-webhook_handlers = WebhookHandlers(
-    product_created=product_created,
-    product_updated=product_updated,
-    product_deleted=product_deleted,
-)
-
-
 app = SaleorApp(
+    manifest=Manifest(
+        name="Sample Saleor App",
+        version="0.1.0",
+        about="Sample Saleor App seving as an example.",
+        data_privacy="",
+        data_privacy_url="http://172.17.0.1:5000/dataPrivacyUrl",
+        homepage_url="http://172.17.0.1:5000/homepageUrl",
+        support_url="http://172.17.0.1:5000/supportUrl",
+        id="saleor-simple-sample",
+        permissions=["MANAGE_PRODUCTS", "MANAGE_USERS"],
+        configuration_url=Manifest.url_for("configuration-form"),
+        extensions=[],
+    ),
     validate_domain=validate_domain,
     save_app_data=store_app_data,
-    webhook_handlers=webhook_handlers,
+    webhook_handlers=WebhookHandlers(
+        product_created=product_created,
+        product_updated=product_updated,
+        product_deleted=product_deleted,
+    ),
     get_webhook_details=get_webhook_details,
+    use_insecure_saleor_http=settings.debug,
```

Comment:
This is to make the development logic more explicit - previously it was dependant on the `settings.debug` value which was very ambigiuos, ideally, I think I'd rather have Saleor share it's full URL with the app (including the schema, and potentially a custom `/graphql/` endpoint)